app/src/main/java/com/hyperion/ths/marvel_03/data/source/remote/api/service/ServiceClient.java

```diff
@@ -0,0 +1,50 @@
+package com.hyperion.ths.marvel_03.data.source.remote.api.service;
+
+import android.app.Application;
+import android.support.compat.BuildConfig;
+import com.hyperion.ths.marvel_03.utils.Constant;
+import java.util.concurrent.TimeUnit;
+import okhttp3.Cache;
+import okhttp3.Interceptor;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by ths on 30/05/2017.
+ */
+
+ class ServiceClient {
```

Comment:
ERROR: 'class def ident' have incorrect indentation level 1, expected level should be 0.